test/Conversion/TorchOnnxToTorch/simple_ops_a_to_f.mlir

```diff
@@ -853,6 +1024,17 @@ func.func @test_concat_3d_axis_negative_3(%arg0: !torch.vtensor<[2,2,2],f32>, %a
   return %0 : !torch.vtensor<[4,2,2],f32>
 }
 
+// -----
+
+// CHECK-LABEL:   func.func @"torch-jit-export-exp"
+func.func @"torch-jit-export-exp"(%arg0: !torch.vtensor<[3,4],f32>) -> !torch.vtensor<[3,4],f32> attributes {torch.onnx_meta.ir_version = 3 : si64, torch.onnx_meta.opset_version = 6 : si64, torch.onnx_meta.producer_name = "pytorch", torch.onnx_meta.producer_version = "0.3"} {
```

Comment:
The name is a bit gross. Change it to something that does not require quotes. Also the following attributes are not required: `torch.onnx_meta.producer_name = "pytorch", torch.onnx_meta.producer_version = "0.3"`